HACKING.md

```diff
@@ -1,4 +1,4 @@
-﻿# Development HOWTO
```

Comment:
Looks like I removed unicode BOM here.

Reply:
Which is weird since this is supposed to be utf8 %)